### PR TITLE
#162199452 feat(trending): Trending articles

### PR DESCRIPTION
**Pull request by wiztemple**
#### What does this PR do?
- Get trending articles
 #### Description of task to be completed
- create a function to remove duplicate arrays
- create a function to return the top six trending articles
- run database seeders for data
- Test feature
#### How can this be manually tested
- on the terminal run git clone https://github.com/andela/haven-ah-backend.git.
- run git checkout ft-get-trending-article-162199452
- ensure that PostgreSQL is running on your local machine.
- set up databases (test and development) following the .env.sample file.
- run npm install to install the npm packages.
- run npm test to run the unit and integration tests.
- To test on postman: run npm run start-dev and open up postman.
#### What are the relevant Pivotal Tracker stories?
[#162199452]()

#### Screenshots(if appropriate)

### CODE REVIEW

File: server/controllers/article.js [hunk 1]
```diff
@@ -3,8 +3,10 @@ import articleRepo from '../repository/articleRepository';
 import { goodHttpResponse, badHttpResponse, paginatedHttpResponse } from '../utilities/httpResponse';
 import tagRepo from '../repository/tagRepository';
 import ratingRepo from '../repository/ratingRepository';
+import reactionRepo from '../repository/reactionRepository';
 import notificationRepo from '../repository/notificationRepository';
 import rankArticles from '../utilities/articlesRanker';
+import removeArrayDuplicates from '../utilities/processArray';
```
Comment (seyi-adeleke) [inline server/controllers/article.js]: This file should be named `removeArrayDuplicates`

**Review by seyi-adeleke — COMMENTED** [review]
What happens if this endpoint returns an empty payload?

File: server/controllers/article.js (hunk 1 above)
Comment (wiztemple) [reply server/controllers/article.js]: This is noted.

**Comment by wiztemple** [comment]
I'm thinking of picking articles from the database if the length of the trending articles is zero.

**Review by seyi-adeleke — CHANGES_REQUESTED** [review]
Your package lock shouldn't have a diff

**Pull request closed without merging**